doc/classes/JSONParseResult.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="JSONParseResult" inherits="Reference" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Data class wrapper for decoded JSON.
 	</brief_description>
 	<description>
+		Returned by [method JSON.parse], [JSONParseResult] contains decoded JSON or error information if JSON source not successfull parsed.	Check if JSON source successfull parsed with [code]if json_result.error == 0[/code].
```

Comment:
"if JSON source was not successfully parsed."
Too many spaces after second sentence.
Last sentence:
"You can check if JSON source was successfully parsed"